test/NuGet.Core.FuncTests/NuGet.XPlat.FuncTest/XPlatPushTests.cs

```diff
@@ -1,28 +1,32 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-using System;
 using System.Collections.Generic;
-using System.Diagnostics;
+using System.Globalization;
 using System.IO;
-using System.Linq;
 using System.Threading.Tasks;
 using NuGet.Configuration;
 using NuGet.Test.Utility;
+using FileSystemUtils = NuGet.Test.Utility.TestFileSystemUtility;
 using Xunit;
 
 namespace NuGet.XPlat.FuncTest
 {
     public class XPlatPushTests
     {
+        private const string Command_Push = "push";
+        private const string Param_ApiKey = "--api-key";
+        private const string Param_ConfigFile = "--config-file";
+        private const string Param_Source = "--source";
+
         [PackageSourceTheory]
         [PackageSourceData(TestSources.MyGet)]
         [PackageSourceData(TestSources.ProGet, Skip = "No such host is known")]
         [PackageSourceData(TestSources.Klondike, Skip = "401 (Invalid API key)")]
         [PackageSourceData(TestSources.NuGetServer, Skip = "No such host is known")]
         public async Task PushToServerSucceeds(PackageSource packageSource)
         {
-            // Arrange
+            // Setup
```

Comment:
https://github.com/NuGet/NuGet.Client/blob/dev/docs/coding-guidelines.md#unit-test-structure

We use Arrange, Act, Assert as the coding guidelines unit test structure. This is known as the AAA pattern. Perhaps not well known (I never heard of it before joining NuGet), but doing a web search in multiple search engines finds many results, so I can't claim it's made up just for NuGet.